From 21d894c6070855dd21adb1d8a4bceeeb0375a236 Mon Sep 17 00:00:00 2001
From: Xiaowen Liu <b37945@freescale.com>
Date: Fri, 19 Jul 2013 17:38:19 +0800
Subject: [PATCH 215/602] ENGR00215174 remove fsl.imx jar from bootclasspath.

remove fsl.imx jar from bootclasspath.
because it will make user build image can't work.

Signed-off-by: Xiaowen Liu <b37945@freescale.com>
---
 imx6/etc/init.rc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/imx6/etc/init.rc b/imx6/etc/init.rc
index a241d6c..7e332e3 100755
--- a/imx6/etc/init.rc
+++ b/imx6/etc/init.rc
@@ -37,7 +37,7 @@ loglevel 3
     export ANDROID_STORAGE /storage
     export ASEC_MOUNTPOINT /mnt/asec
     export LOOP_MOUNTPOINT /mnt/obb
-    export BOOTCLASSPATH /system/framework/core.jar:/system/framework/core-junit.jar:/system/framework/bouncycastle.jar:/system/framework/ext.jar:/system/framework/framework.jar:/system/framework/telephony-common.jar:/system/framework/mms-common.jar:/system/framework/android.policy.jar:/system/framework/services.jar:/system/framework/apache-xml.jar:/system/framework/fsl.imx.jar
+    export BOOTCLASSPATH /system/framework/core.jar:/system/framework/core-junit.jar:/system/framework/bouncycastle.jar:/system/framework/ext.jar:/system/framework/framework.jar:/system/framework/telephony-common.jar:/system/framework/mms-common.jar:/system/framework/android.policy.jar:/system/framework/services.jar:/system/framework/apache-xml.jar
     export SHM_MOUNTPOINT /mnt/shm
     export LIGHTSENSOR_PATH /sys/class/i2c-adapter/i2c-1/1-0044
     export VPU_FW_PATH /system/lib/firmware/vpu
-- 
1.8.0

